contracts/Pod.sol

```diff
@@ -446,15 +441,18 @@ contract Pod is
      * @dev Claim POOL for PrizePool Pod and adds/transfers those token to the Pod TokenDrop smart contract.
      * @return uint256 claimed amount
      */
-    function claimPodPool() public returns (uint256) {
+    function claimPodReward() public returns (uint256) {
+        // Claim outstanding reward amount from PrizePool TokenFaucet
         faucet.claim(address(this));
-        uint256 balance = pool.balanceOf(address(this));
+
+        // Check Current Pod reward balance
+        uint256 balance = reward.balanceOf(address(this));
```

Comment:
Here is an optimization for you:

Notice how `reward` is being referenced twice in this function?  This results in two SLOADs!  It's actually cheaper if you assign it to a local var and then reference that.

Reply:
Ohhh good to know - I figured because it would a global constant in the contract that wouldn't cause an extra SLOAD.

Thank you sensei 🙇‍♂️